app/helpers/foreman_ansible/ansible_roles_helper.rb

```diff
@@ -31,6 +31,12 @@ def ansible_proxy_import(hash)
                            ansible_proxy_links(hash))
     end
 
+    def vcs_import
+      select_action_button("",
```

Comment:
We should not display this button if the user does not have the required permissions.

Reply:
Missed this as well. Will add later.

Reply:
Actually, I am not sure how to handle this... Ideally, this would be handled on the React side.